### ISSUE TITLE
Error with HA 2022.03 update

### ISSUE THREAD

**Issue by Krojack** [ben8p/lovelace-auto-reload-card#6]
Started getting the following error after updating to the March HA build.

2022-03-03 18:08:05 ERROR (MainThread) [frontend.js.latest.202203010] https://homeassistant.lan:<port>/hacsfiles/lovelace-auto-reload-card/auto-reload-card.js:29:32 Uncaught TypeError: Cannot read properties of null (reading 'querySelector')

**Comment by Protoncek** [comment]
Yep. And it doesn't update lovelace anymore. Please fix it... it's very usefull card.

**Comment by adamfreefall** [comment]
Temporary fix:
In file: auto-reload-card.js
replace line:
const refresh = buttonMenu.querySelector('[aria-label=Refresh]');
with:
const refresh = null;

**Comment by kwithus** [comment]
Where do I find this file? 
will there be a permanent fix?

**Comment by ben8p** [comment]
Hi,
Thanks you for reporting.
I don't have enough time right now to investigate and make a fix but if anyone wants to make a PR, I'd be happy to merge it

**Comment by rjenx** [comment]
Uninstalled the component until a fix is available and still getting the error, I guess somewhere there is residual code.

**Comment by ben8p** [comment]
I think Lovelace has a cache. You need to refresh it

**Comment by zSprawl** [comment]
This issue is still present in 2022.7 but at least the temp fix above appears to be working by basically removing the button and just leveraging the javascript.refresh() part of the if statement. Hopefully that is enough.

Another option I've heard people mention is to use Fully Kiosk mode, but that comes with a lot of other features that likely need setting up. I like this approach if it sticks.

Oh and do a full reboot of everything, and even go into your dashboard and re-save the page with the card to ensure the js takes.

**Comment by zSprawl** [comment]
Any ideas how you clear this cache? 

I used the SSH addon and found the auto-reload-card.js file in config/www/custom_components/lovelace-auto-reload-card/ and I've edited it with the fix above.

I have obviously rebooted and of course cleared the browser local cache. 

I can even go to the url of:

http://192.168.100.69:8123/hacsfiles/lovelace-auto-reload-card/auto-reload-card.js

And here I can clearly see the fix mentioned above has not been applied to the expected file.

**Comment by zSprawl** [comment]
In the meantime, I was able to copy the file to a location I could reach, change the resource link, and reboot to validate the fix does indeed work.

Also in reflection, I think the error only appeared when the tablet user fires off (which doesn't have admin rights). I don't know if that could be related.
